packages/svelte-undp-design/src/lib/Carousel/Carousel.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="ts">
+	import Carousel from 'svelte-carousel';
+
+	let carousel: Carousel;
+
+	const handlePrevClick = () => {
+		carousel.goToPrev();
+	};
+	const handleNextClick = () => {
+		carousel.goToNext();
+	};
```

Comment:
@lrt-sibat could you update `FluidCarousel.svelte` instead of adding this one?

https://github.com/UNDP-Data/geohub/blob/feat/ceei-dashboard/packages/svelte-undp-design/src/lib/FluidCarousel/FluidCarousel.svelte

your new `Carousel` component looks almost same with FluidCarousel.

Reply:
@lrt-sibat I think your new Crousel.svelte is not used anywhere, right? please remove it if it is not used.